Home page updates

insmac opened the pull request:
Screen recording
https://user-images.githubusercontent.com/1871646/135637477-f796376d-fabb-4385-956e-0b324d996194.mp4


Updates

- Added Metabase logo to Integrations, shrunk down size of the logos overall
- Made company logos in the top consistent not to exceed the container width
- Adjust feature breakdown columns (_Built for performance_, _Optimized for time series_, etc) to be in line vertically on smaller screens
- Introduce `<SvgImage />` component for inline SVGs (natively supported by Docusaurs - [Inline SVGs](https://docusaurus.io/docs/markdown-features/assets#inline-svgs)
- Optimize the home page by inlining SVGs across, decreasing asset number to load

bsmth reviewed (approved):
LGTM 👍🏻

bluestreak01 commented:
We got a consistent new warning on this new page (front page):

![image](https://user-images.githubusercontent.com/7276403/135659628-91d7a170-d808-4850-a94e-544ba680b144.png)

vs existing page:

![image](https://user-images.githubusercontent.com/7276403/135659689-7db98b4d-0772-4087-8480-35aa1b8c1521.png)

bluestreak01 reviewed (changes requested):
Can performance warning be corrected ?

insmac commented:
@bluestreak01 That's probably because images are inline and a few of them have complex structure (mostly the tree big icons. I'll find a workaround for this.

insmac commented:
Updated Lighthouse audit:
<img width="1036" alt="CleanShot 2021-10-04 at 10 58 25@2x" src="https://user-images.githubusercontent.com/1871646/135823192-d14df626-a466-4775-aee8-ce66ddf2497c.png">

@bluestreak01 Please re-run Lighthouse audit on your end, I have changed SVG illustrations back to being loaded as assets which fixed the problem.
